@AH-64D Apache Longbow/Addons/fza_ah64_systems/functions/apu/fn_apu.sqf

```diff
@@ -0,0 +1,53 @@
+/* ----------------------------------------------------------------------------
+Function: fza_systems_fnc_systemsAPU
+
+Description:
+	Defines key values for the simulation.
+
+Parameters:
+	_heli - The helicopter to get information from [Unit].
+
+Returns:
+	...
+
+Examples:
+	...
+
+Author:
+	BradMick
+---------------------------------------------------------------------------- */
+params ["_heli", "_deltaTime"];
+#include "\fza_ah64_systems\headers\systems.hpp"
+
+private _apuBtnOn   = _heli getVariable "fza_systems_apuBtnOn";
+private _battBusOn  = _heli getVariable "fza_systems_battBusOn";
+private _apuOn      = _heli getVariable "fza_systems_apuOn";
+private _apuDamage     = _heli getHitPointDamage "hit_apu";
+private _apuStartDelay = _heli getVariable "fza_systems_apuStartDelay";
+private _apuRPM_pct    = _heli getVariable "fza_systems_apuRPM_pct";
+private _apuFF_kgs     = 0.0;
+
+if (_apuBtnOn && _battBusOn) then {
```

Comment:
Eventually I would like this to not be a perframe function, but work on timers and eventt handlers. However for now this is okay